tests.js

```diff
@@ -564,6 +590,31 @@ window.Specs = {
 		}
 	},
 
+
+	"css2-media": {
+		"title": "CSS 2 Media types",
+		"links": {
+			"tr": "CSS22/media.html",
+			"dev": "css2/"
+		},
+		"status": {
+			"stability": "stable",
+			"first-snapshot": 1998,
+			"last-snapshot": 1998
+		},
+		"@rules": {
+			"@media": {
+				"links": {
+					"tr": "#at-media-rule",
+					"dev": "#at-media-rule"
+				},
+				"tests": [
+					"@media print { color: green; }", "@media print, screen { color: green; }"
```

Comment:
I think `color` is not a valid descriptor, so you should include some selector inside the rules.

Also i wonder if we should test the other media types (`all`, `braille`...).
I see that CSS2 defines unknown types should be ignored without invalidating the rule, and also that most of the types are actually deprecated in Media Queries 4, so i'm not sure.

Reply:
Oh, right! I missed that I have to add a style rule for testing. Surprisingly, it was still parsed as valid rule.

I intentionally didn't add the other media types. Browsers are required to support them, though there is no value for authors if we test for their support because they are silently ignored.